### ISSUE TITLE
Table of contents: "Layout" is not a valid block category

### ISSUE THREAD

**Issue by carolinan** [WordPress/gutenberg#60930]
### Description

The block.json schema lists the valid block categories as:
```
"text",
"media",
"design",
"widgets",
"theme",
"embed"
```

But the experimental table of content block uses the category "layout" in block.json.

Either this category needs to be updated in the block, or it needs to be added to the schema.

### Step-by-step reproduction instructions

View the source of the block.json file.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by colorful-tones** [comment]
I just updated the Issue label for the Table of Content block, because I noticed it did not have the “(experimental)” at the end.

I would say that this block could go under the “text” category. 

Probably a terrible idea for maintaining things, but I wonder if an “experimental” category should be added. Then this block and Form block could both get categorized as “experimental.” Again, probably a bad idea.
